tests/unittest/_torch/modeling/test_modeling_mamba_hybrid.py

```diff
@@ -1,166 +0,0 @@
-import unittest
-from copy import deepcopy
-
-import torch
-
-import tensorrt_llm
-from tensorrt_llm._torch.attention_backend.utils import get_attention_backend
-from tensorrt_llm._torch.metadata import KVCacheParams
-from tensorrt_llm._torch.model_config import ModelConfig
-from tensorrt_llm._torch.models.modeling_mamba_hybrid import (
-    MambaHybridConfig, MambaHybridForCausalLM)
-from tensorrt_llm._torch.pyexecutor.resource_manager import KVCacheManager
-from tensorrt_llm.bindings.executor import KvCacheConfig
-from tensorrt_llm.mapping import Mapping
-
-MAMBA_HYBRID_CONFIG = {
-    "architectures": ["MambaHybridForCausalLM"],
-    "attention_bias": False,
-    "attention_dropout": 0.0,
-    "attention_head_dim": 128,
-    "bos_token_id": 1,
-    "chunk_size": 256,
-    "conv_kernel": 4,
-    "eos_token_id": 2,
-    "expand": 2,
-    "hidden_dropout": 0.0,
-    "hidden_size": 4096,
-    "hybrid_override_pattern":
-    "M-M-M-M*-M-M-M-M-M*-M-M-M-M-M*-M-M-M-M-M*-M-M-M-M-M-",
-    "initializer_range": 0.02,
-    "intermediate_size": 21504,
-    "layer_norm_epsilon": 1e-05,
-    "mamba_d_state": 128,
-    "mamba_head_dim": 64,
-    "mamba_hidden_act": "silu",
-    "mamba_num_heads": 128,
-    "mamba_proj_bias": False,
-    "max_position_embeddings": 8192,
-    "mlp_bias": False,
-    "mlp_hidden_act": "relu2",
-    "model_type": "mamba_hybrid",
-    "n_groups": 8,
-    "num_attention_heads": 32,
-    "num_hidden_layers": 52,
-    "num_key_value_heads": 8,
-    "num_logits_to_keep": 1,
-    "pad_token_id": 0,
-    "rescale_prenorm_residual": True,
-    "residual_in_fp32": True,
-    "rms_norm_eps": 1e-05,
-    "sliding_window": None,
-    "ssm_states_size": 128,
-    "tie_word_embeddings": False,
-    "time_step_floor": 0.0001,
-    "time_step_max": 0.1,
-    "time_step_min": 0.001,
-    "time_step_rank": 256,
-    "torch_dtype": "bfloat16",
-    "transformers_version": "4.48.0.dev0",
-    "use_bias": False,
-    "use_cache": True,
-    "use_conv_bias": True,
-    "use_mamba_kernels": True,
-    "vocab_size": 131072,
-}
-
-
-class TestMambaHybrid(unittest.TestCase):
```

Comment:
I think we can keep it as a sanity check test for Nemotron-H model.

Reply:
Done.